apps/kol-amelamdim/pages/api/get-active-weekly-article.ts

```diff
@@ -0,0 +1,18 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { WeeklyArticle } from '@kol-amelamdim/models';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  try {
+    const activeArticle = await WeeklyArticle.findOne({
+      isActiveArticle: true,
+    });
+    if (activeArticle) {
+      return res.status(200).json(activeArticle);
+    }
+  } catch (e) {
+    return res.status(404);
```

Comment:
i thing we should keep our nice convention with the `return res.status(400).json(errorKuku);`